backend/v3/storage/database/dialect/postgres/migration/011_session_table/up.sql

```diff
@@ -0,0 +1,93 @@
+CREATE TABLE zitadel.session_user_agents (
+    instance_id TEXT
+    , fingerprint_id TEXT CHECK (fingerprint_id <> '')
```

Comment:
just to improve readability
```suggestion
    instance_id TEXT NOT NULL
    , fingerprint_id TEXT NOT NULL CHECK (fingerprint_id <> '')
```